pusher-dotnet-client.nuspec

```diff
@@ -1,26 +1,26 @@
-<?xml version="1.0"?>
-<package >
-  <metadata>
-    <id>PusherClient</id>
-    <version>$version$</version>
-    <title>Pusher .NET Client Library</title>
-    <authors>Pusher, Phil Leggetter</authors>
-    <owners>Pusher</owners>
-    <licenseUrl>https://github.com/pusher/pusher-dotnet-client/blob/master/LICENSE.txt</licenseUrl>
-    <projectUrl>https://github.com/pusher/pusher-dotnet-client</projectUrl>
-    <requireLicenseAcceptance>false</requireLicenseAcceptance>
-    <description>The .NET library for interacting with the Pusher WebSocket API.
-Register at http://pusher.com.</description>
-    <releaseNotes></releaseNotes>
-    <copyright>Copyright 2015</copyright>
-    <tags>pusher realtime websocket</tags>
-    <dependencies>
-        <dependency id="Newtonsoft.Json" version="7.0.1" />
-        <dependency id="WebSocket4Net" version="0.13.1" />
-    </dependencies>
-  </metadata>
-  <files>
-    <file src="..\README.md" target="." />
-    <file src="..\CHANGELOG.md" target="." />
-  </files>
-</package>
\ No newline at end of file
+<?xml version="1.0"?>
+<package >
+  <metadata>
+    <id>PusherClient</id>
+    <version>$version$</version>
+    <title>Pusher .NET Client Library</title>
+    <authors>Pusher, Phil Leggetter</authors>
+    <owners>Pusher</owners>
+    <licenseUrl>https://github.com/pusher/pusher-dotnet-client/blob/master/LICENSE.txt</licenseUrl>
+    <projectUrl>https://github.com/pusher/pusher-dotnet-client</projectUrl>
+    <requireLicenseAcceptance>false</requireLicenseAcceptance>
+    <description>The .NET library for interacting with the Pusher WebSocket API.
+Register at http://pusher.com.</description>
+    <releaseNotes></releaseNotes>
+    <copyright>Copyright 2015</copyright>

```

Comment:
How do licences work? Are you always supposed to have them read the current year or what...?